ngraph/python/BUILDING.md

```diff
@@ -56,7 +64,13 @@ adjust the number of threads used in the building process to your machine's capa
 
     `cmake --build . --target python_wheel --config Release -j 8`
 
-The final wheel should be located in `build\python\dist` directory.
+The final wheel should be located in `ngraph\python\dist` directory.
+
+4. Configure the environment for the Inference Engine Python API:
+
+    `call <CMAKE_INSTALL_PREFIX>\bin\setupvars.bat`
+
+NOTE: This step is not required for using alone nGraph Wheel, but it needed for most usage scenarios (for example for run unit tests).
```

Comment:
Could you please change the note to the following?
```
> **NOTE**: Skip this step if you want to use the nGraph Wheel by itself. This step is required for most usage scenarios, like running unit tests.
```

Reply:
done